client/src/Constants.js

```diff
@@ -0,0 +1,10 @@
+/* Store constant values here */
+
+export const minPasswordLength = 6;
+export const minPasswordErrMsg = `Your password must be at least ${minPasswordLength} characters long`;
+export const takenEmailErrMsg = "That email is already taken";
+
+const serverUrl = "http://localhost:5000/";
+const clientUrl = "http://localhost:3000/";
+export const serverSignUpUrl = serverUrl + "user";
+export const clientDashboardUrl = clientUrl + "dashboard";
```

Comment:
I like the idea of storing the routes, but how do you plan to handle more complicated backend urls like `users/{userId}` ?
I also think you should store the frontend routes in a separate file.

Reply:
Maybe by using a function that takes in the user Id as a paramater. I will separate the frontend and backend routes into two separate files in a future cleanup PR.